vmm/src/device_manager.rs

```diff
@@ -450,6 +450,50 @@ impl DeviceRelocation for AddressManager {
                         .register_ioevent(event, &io_addr, NoDatamatch)
                         .map_err(|e| io::Error::from_raw_os_error(e.errno()))?;
                 }
+            } else {
+                let virtio_dev = virtio_pci_dev.virtio_device();
+                let mut virtio_dev_locked = virtio_dev.lock().unwrap();
```

Comment:
I don't really like the `_locked` just replace the `virtio_dev` variable.

Reply:
Yes I changed it!